plugins/github-issues/src/api/gitHubIssuesApi.ts

```diff
@@ -151,20 +189,50 @@ export const gitHubIssuesApi = (
   return { fetchIssuesByRepoFromGitHub };
 };
 
+function formatFilterValue(value: IssuesFilters[keyof IssuesFilters]): string {
+  if (Array.isArray(value)) {
+    return `[ ${value.map(formatFilterValue).join(', ')}]`;
+  }
+
+  return typeof value === 'string' ? `\"${value}\"` : `${value}`;
+}
+
+function createFilterByClause(filterBy?: IssuesFilters): string {
```

Comment:
consider this optional as there are some tests but I would recommend adding some unit tests for this specific function to ensure it works as intended given that there are quite a few if statements inside it and that there's some ```typeof value === 'string' ? `\"${value}\"` : `${value}``` which I don't know if it's captured by the existing tests.

You can add test by exporting it with `export function createFilterByClause` but you don't have to add an export in the index.ts file.